crates/t-digest/src/lib.rs

```diff
@@ -843,9 +843,7 @@ mod tests {
     fn test_merge_sorted_against_skewed_distro() {
         let t = TDigest::new_with_size(100);
         let mut values: Vec<f64> = (1..=600_000).map(f64::from).collect();
-        for _ in 0..400_000 {
-            values.push(1_000_000.0);
-        }
+        values.resize(1_000_000, 1_000_000.0);
```

Comment:
This line (and the other identical one below) should be
```rust
        values.resize(400_000, 1_000_000.0);
```
since the first argument is the new length.

Reply:
I got the `1_000_000` (for the first argument) from the `400_000` loop range + the `600_000` from the current length of `values` created in the line above.
 
maybe something like this would be a better fix?
```rust
         let mut values: Vec<f64> = (1..=600_000).map(f64::from).collect();
         values.resize(values.len() + 400_000, 1_000_000.0);
```

Reply:
yeah, that looks a lot cleaner